src/main/java/eu/elixir/ega/ebi/it/res/ResBase.java

```diff
@@ -0,0 +1,34 @@
+package eu.elixir.ega.ebi.it.res;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.http.ContentType;
+import com.jayway.restassured.specification.RequestSpecification;
+
+public class ResBase {
+    RequestSpecification REQUEST;
+    String baseURI;
+    int port;
+    String unencryptedChecksum;
+    String fileId;
+
+    public ResBase() {
```

Comment:
The spacing inside the ResBase seems to be double compared to the other ones

Reply:
This comment is not clear.

Reply:
yes sorry, in the resBase there are 8 spaces after the Resbase() class but for the others there are just 4 spaces

Reply:
oh ok, formatting was missing. Done.